frontend/views/containers/user-settings/AppLogs.vue

```diff
@@ -82,6 +85,16 @@ export default {
         .filter(({ type }) => this.form.filter.includes(type))
         .map(({ type, msg, timestamp }) => `${timestamp} [${type}] ${msg.map(JSON.stringify).join(' ')}`)
         .join('\n')
+    },
+    instructions () {
+      const errorMsg = this.$route.query.errorMsg
+      const linkTag = {
+        'a_': '<a class="link" target="_blank" href="https://github.com/okTurtles/group-income-simple/issues">',
+        '_a': '</a>'
+      }
+      return errorMsg
+        ? L('You faced a recent error: "{errorMsg}". Please download the logs and {a_}send to us{_a}, so that we can help you solve the problem.', { errorMsg, ...linkTag })
+        : L('If you are facing any problem with Group Income, download the logs and {a_}send to us{_a}, so that we can help you solve the problem.', linkTag)
```

Comment:
Minor wording change:

- `You faced a recent error: "{errorMsg}". Please download the logs and {a_}send to us{_a}, so that we can help you solve the problem.` => `Recent error: "{errorMsg}". Please download the logs and {a_}send them to us{_a} so we can help troubleshoot.`
- `If you are facing any problem with Group Income, download the logs and {a_}send to us{_a}, so that we can help you solve the problem.` => `If you encounter problems, please download the logs and {a_}send them to us{_a}.`